.github/workflows/update-as.yml

```diff
@@ -0,0 +1,43 @@
+name: Create a PR to build the app using newest a-s version available
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ./test-fixtures/requirements.txt
+    - name: Modify carthage 
```

Comment:
We're just touching the Cartfile, so I would rename this to Modifying Cartfile

Reply:
oh right, don't know why I wrote carthage instead of Cartfile... changing that... thanks